networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -9,28 +9,20 @@ type Mutation {
   )
 }
 
-input CreateGameSessionFromTemplateInput {
-  gameTemplateId: ID!
-}
-
-# define a GameTemplate in amplify and graphql to list all gametemplates sorted by updatedAt
-type GameTemplate @model
+type PublicGameTemplate @model
 @auth(
   rules: [
     { allow: public, operations: [read], provider: iam  },
-    { allow: private, operations: [read, create, update, delete], provider: userPools }
+    { allow: owner, provider: userPools },
+    { allow: private, operations: [read], provider: userPools },  
   ]
 )
 @aws_iam
 @aws_cognito_user_pools
 {
   id: ID!
+  owner: String
   title: String!
-  owner: String! @index(
```

Comment:
indexing off owner blows up a lot of the background amplify processes, and isn't required as the @auth params specified above will take care of the query results etc

Reply:
More clearly, we won't need to explicitly return values by owner in central, because we are using the @auth operations (read, write, etc) to do that for us.